tests/Fist/regtest-spgr/TEST_FILES

```diff
@@ -0,0 +1,8 @@
+# runs are executed in the same order as in this file
+# the second field tells which test should be run in order to compare with the last available output
+# e.g. 0 means do not compare anything, running is enough
+#      1 compares with the space group name and number of operations provied in the SPGR section of the output file
+#      for details see cp2k/tools/do_regtest
+zro.inp                                      0
```

Comment:
Do you still want the test to check that the correct space group was recognized?

Reply:
@oschuett I would like to but there is no function in spglib_f08 module to get the space group number. The only way would be to get the dataset. However, despite both the dataset structure and dataset get function being defined in the module and being made public, compilation fails saying these cannot be found. So, I can only get the various space group names which are all strings. I don't know, considering the structure of the regtests, if it is possible to compare strings.

Reply:
According to the [docs](https://spglib.github.io/spglib/api.html#spg-get-international) the routine `spg-get-international` returns the space group number. So, you should already have it in `ierr` in your [code](https://github.com/cp2k/cp2k/blob/5095393704225a9e265ec3c0f80189f28470a976/src/motion/space_groups.F#L245).

Reply:
Brilliant. I did not realise. That's fixed too. Now, how do I make it so that the regtest compare this value? Which number should I put instead of 0 in the second column of the TEST_FILES?

Reply:
You need to add a new pattern to [TEST_TYPES](https://github.com/cp2k/cp2k/blob/master/tests/TEST_TYPES). The format is described at the end of that file. Don't forget to increment the counter at the top. Afterwards you can put `93` instead of `0` in the second column of your TEST_FILES.